shopfloor/__manifest__.py

```diff
@@ -27,6 +27,7 @@
         #  OCA / stock-logistics-workflow
         "stock_quant_package_dimension",
         "stock_quant_package_product_packaging",
+        "stock_picking_completion",
```

Comment:
Where is this module?
Rename to stock_picking_progress

Reply:
done.